index.ts

```diff
@@ -609,12 +609,25 @@ export default class FreeSound {
    * downloading sounds.
    *
    * This method can set both kinds of tokens
+   * ```typescript
+   * await freeSound.setToken('your-api-key', 'oauth');
+   * ```
    */
   setToken(token: string, type?: "oauth"): string {
     this.authHeader = `${type === 'oauth' ? 'Bearer' : 'Token'} ${token}`;
     return this.authHeader;
   }
 
+  /**
+   * ```typescript
```

Comment:
Convention is for example to follow doc comments. Here's a few examples:
* https://github.com/lodash/lodash/blob/master/escape.js#L14-L40
* https://github.com/lodash/lodash/blob/master/add.js#L3-L16

Note that they're always separated by a new line as well. This significantly helps with readability.

Reply:
Ah, I see. Thank for the links to the examples. They're very instructive.